compiler/ballerina-parser/src/main/java/io/ballerina/compiler/internal/parser/tree/STClientDeclarationNode.java

```diff
@@ -0,0 +1,124 @@
+/*
+ *  Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
```suggestion
 *  Copyright (c) 2022, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```
Should be updated in other generated files.

Reply:
Done, I originally assumed we were keeping it as is since every time treegen is run it'll update this file.